src/code/z_lifemeter.c

```diff
@@ -172,18 +249,24 @@ s32 func_80078E84(GlobalContext* globalCtx) {
     } else {
         interfaceCtx->unk_228 = 0x140;
         interfaceCtx->unk_226 -= 0x10;
-
         if (interfaceCtx->unk_226 <= 0) {
             interfaceCtx->unk_226 = 0;
             globalCtx->damagePlayer(globalCtx, -(gSaveContext.health + 1));
             return 1;
         }
     }
-
     return 0;
 }
 
 void Health_Draw(GlobalContext* globalCtx) {
+    static u8* sHeartTextureList[] = {
+        0x02000400, 0x02000100, 0x02000100, 0x02000100, 0x02000100, 0x02000100, 0x02000200, 0x02000200,
+        0x02000200, 0x02000200, 0x02000200, 0x02000300, 0x02000300, 0x02000300, 0x02000300, 0x02000300,
+    };
+    static u8* sDoubleDefenseHeartTextureList[] = {
```

Comment:
This DoubleDefense can probably also be shortened to DD. And ideally the order of DD and Heart should be consistent throughout the file (mostly referring to color and factor arrays which are currently HeartDD, so this one should probably be HeartDD as well)